### DISCUSSION TITLE
Advanced Netconf Explorer

### DISCUSSION THREAD

**Discussion by minexn** [kernelkit/infix#467] [General]
I'm testing Infix v24.04.2 with Advanced Netconf Explorer.

On connection, I'm getting the following error.

```
admin@ca-mx18-dev-1:/> show follow 
Press Ctrl-C to abort ────────────────────────────────────────────
Apr 23 10:34:03 ca-mx18-dev-1 netopeer2-server[837]: Failed to set hostkey "genkey" (/tmp/Gvfcli).
Apr 23 10:34:03 ca-mx18-dev-1 netopeer2-server[837]: Failed to set hostkey "genkey" (/tmp/KhjmdK).
```

Any ideas?

**Comment by troglobit** [comment]
Hi, none of us have used anx before, but I tried setting it up using [this blog post](https://www.linkedin.com/pulse/using-advanced-netconf-explorer-anx-syed-hassan/).  I cannot reproduce your particular issue, for me it halts on:

```
May 30 04:16:15 infix-00-00-00 netopeer2-server[2755]: SSH key exchange error (Socket error: disconnected).
May 30 04:16:15 infix-00-00-00 netopeer2-server[2755]: SSH key exchange error (kex error : no match for method server host key algo: server [rsa-sha2-512,rsa-sha2-256], client [ssh-rsa,ssh-dss,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521]).
```

 - Did anx work with Infix prior to v24.04.2?
 - Did you upgrade from a previous version of Infix?
 - Can you connect with any other NETCONF client, e.g. netopeer2-cli?
